geometric/optimize.py

```diff
@@ -872,8 +872,8 @@ def Optimize(coords, molecule, IC, engine, dirname, params, xyzout=None, xyzout2
 
     Returns
     -------
-    np.ndarray
-        Nx3 array of optimized Cartesian coordinates in atomic units
+    progress: Molecule
```

Comment:
Here I changed the return value of Optimize to be a Molecule class object `progress` which contains the geometries and energies along the optimization. This is not used directly in run_json.py, but maybe useful later.